libs/core/src/lib/file-uploader/file-uploader.component.ts

```diff
@@ -129,12 +129,26 @@ export class FileUploaderComponent implements ControlValueAccessor, OnInit, OnDe
     @Input()
     inputHidden = false;
 
+    /** Specifies if input is focused from outside the component */
+    @Input() set inputFocused(focused: boolean) {
+        this._inputFocused = focused;
+        if (focused) {
+            this.inputRefText.nativeElement.focus();
+        }
+    }
+
+    get inputFocused() {
+        return this._inputFocused;
+    }
+
```

Comment:
I'm wondering why should developers manually manage focus of the input? 

Also, I'm reading the a11y requirement
![image](https://user-images.githubusercontent.com/33101123/147681524-4d153fa9-6dbe-4e84-af4b-43901f20d5dc.png)
and I think the only thing you should  actually do is make the input focusable when tabbed over